tox.ini

```diff
@@ -24,73 +21,64 @@ package=wheel
 
 [testenv:docs]
 recreate = True
-deps =
-    -r {toxinidir}/docs-requirements.txt
```

Comment:
q: since this was extracted to `dev-docs` but `readthedocs` still uses the separate `docs-requirements.txt` - is there a way to update that to use `dev-docs` too? Otherwise we may risk having two dependency sets if one gets updated and the other not

Reply:
Great catch! I totally missed that. Updated based on https://docs.readthedocs.io/en/stable/config-file/v2.html#packages